web/src/main/java/org/hillview/dataStructures/PrivateHistogram.java

```diff
@@ -21,11 +21,14 @@
     // TODO(pratiksha): compute the missing value confidence
     private int missingConfidence;  // Confidence for the missing value
     private final double epsilon;
+    private final int columnIndex;
```

Comment:
The histogram is serialized to the UI, we should not store in it fields that are not needed by the UI.
We'll have to dismantle this class and build it more similar to the Heatmap instead.